test/e2e/network/network_policy.go

```diff
@@ -122,7 +128,56 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			})
 		})
 
-		It("should enforce policy based on NamespaceSelector [Feature:NetworkPolicy]", func() {
+		It("should enforce policy, to allow traffic from pods within a namespace and deny ingress traffic for different namespace, based on PodSelector [Feature:NetworkPolicy]", func() {
```

Comment:
This could just be added as an additional `testCannotConnect` inside the existing `It("should enforce policy based on PodSelector [Feature:NetworkPolicy]"`

Reply:
Acknowledged. I have included this test in the existing It().